pkgs/tools/typesetting/tex/pplatex/default.nix

```
@@ -0,0 +1,49 @@
+{
+	pkgs,
+	fetchFromGitHub,
+	stdenv,
+	cmake,
+	pkgconfig,
+	pcre,
+	...
+} :
+	stdenv.mkDerivation {
+		pname = "fop";
+		version = "1.0-rc2";
+
+		src = fetchFromGitHub {
+			owner = "stefanhepp";
+			repo = "pplatex";
+			rev = "25bf7e121178f1fee2452b4fa9961248a045a387";
+			sha256 = "0xw7nvi2l15iyp9sm8vmmqghi54v99bcivqvx89f5v2gw0kw47k3";
+		};
+
+		nativeBuildInputs =
+		[
+			cmake
+			pkgconfig
+		];
+
+		buildInputs =
+		[
+			pcre
+		];
+
+		installPhase = ''
+			mkdir -p $out/bin
+			cp src/pplatex $out/bin
+		'';
+		
+
+		meta = {
+			description = "TeX/LaTeX family error parser";
+			longDescription = ''
+				A program that displays errors from TeX/LaTeX family programs in a user friendly format. Runs with any tex edition.
+			'';
+			homepage = https://github.com/stefanhepp/pplatex;
```

```suggestion
			homepage = "https://github.com/stefanhepp/pplatex";
```
see https://github.com/NixOS/rfcs/pull/45

---

Thank you, I wasn't aware of the changes of rfc 45.